docs/en/serverless/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,66 @@
+* `Instrumentation` object for applications https://opentelemetry.io/docs/kubernetes/operator/automatic/[auto-instrumentation].
+
+For a more detailed description of the components and advanced configuration, refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md[elastic/opentelemetry] GitHub repository.
+
+[discrete]
+== Prerequisites
+
+* An {obs-serverless} project. To learn more, refer to <<observability-create-an-observability-project>>.
+* A running Kubernetes cluster (v1.23 or newer).
+* https://kubernetes.io/docs/reference/kubectl/[Kubectl].
+* https://helm.sh/docs/intro/install/[Helm].
+* (optional) https://cert-manager.io/docs/installation/[Cert-manager], if you opt for automatic generation and renewal of TLS certificates.
+
+[discrete]
+== Collect your data
+
+. <<observability-create-an-observability-project,Create a new {obs-serverless} project>>, or open an existing one.
+. In your {obs-serverless} project, go to **Add Data**.
+. Under **What do you want to monitor?** select **Kubernetes**, and then select **OpenTelemetry: Full Observability**.
++
+[role="screenshot"]
+image::images/quickstart-k8s-otel-entry-point.png[Kubernetes-OTel entry point]
+
+. Follow the on-screen instructions to install all needed components.
++
+[NOTE]
+====
+The default installation deploys the OpenTelemetry Operator with a self-signed TLS certificate valid for 365 days. This certificate **won't be renewed** unless the Helm Chart release is manually updated. Refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md#cert-manager[cert-manager integrated installation] guide to enable automatic certificate generation and renewal using https://cert-manager.io/docs/installation/[cert-manager].
+====
++
+Deploy the OpenTelemetry Operator and EDOT Collectors using the kube-stack Helm chart with the provided `values.yaml` file. You will run a few commands to:
++ 
+ * Add the helm chart repository needed for the installation.
+ * Create a namespace.
+ * Create a secret with an API Key and the {es} endpoint to be used by the collectors.
+ * Install the `opentelemetry-kube-stack` helm chart with the provided `values.yaml`.
+ * Optionally, for instrumenting applications, apply the corresponding `annotations` as shown in {kib}.
+
+[discrete]
+== Visualize your data
+
+After installation is complete and all relevant data is flowing into Elastic,
+the **Visualize your data** section provides a link to the *[OTEL][Metrics Kubernetes]Cluster Overview* dashboard used to monitor the health of the cluster.
+
+[role="screenshot"]
+image::images/quickstart-k8s-otel-dashboard.png[Kubernetes overview dashboard]
+
+[discrete]
+== Troubleshooting and more
+
+* For troubleshooting deployment and installation, refer to https://github.com/elastic/opentelemetry/tree/main/docs/kubernetes/operator#installation-verification[installation verification].
```

Comment:
```suggestion
* To troubleshoot deployment and installation, refer to https://github.com/elastic/opentelemetry/tree/main/docs/kubernetes/operator#installation-verification[installation verification].
```

Reply:
thanks! I'll have to change that too in the stateful guide :)